src/components/Calculator.js

```diff
@@ -3,19 +3,17 @@ import './Calculator.css';
 import calculate from '../logic/calculate';
 
 const Calculator = () => {
-  const [total, setTotal] = useState(null);
-  const [next, setNext] = useState(null);
+  const [state, myState] = useState({ total: null, next: null, operation: null });
 
   const changeState = (e) => {
```

Comment:
Could this be coming from my end 🤦, your app seems not to work properly, it takes 2-3 clicks for a successful click.
i.e I have to click the button 3 times.
And also I can't perform a successful operation like `5+5`, I got the following error 👇 

![image](https://user-images.githubusercontent.com/31897434/130230650-8dafaf7d-fae6-497b-b710-0a009ce980e9.png)